laser_simulator/PoseStampedNamed[] poses
